### PR TITLE
Clicking on a sentence moves tracker to it

### PR DESCRIPTION
**Pull request by benfrancis314**

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -10,61 +11,65 @@ mark {
 
 /* SHADOW, HIGHLIGHTER, and KEYWORD CSS */
 .sentenceHighlighterBlueShadowBlue {
-	background-color:rgb(230, 239, 253); /* Light Blue highlighter */
+	/* TODO: Integrate different color scheme. Keeping values here for near-term future use */
+	background-color:rgb(230, 239, 253) !important; /* Light Blue highlighter */
```
Comment (sjonany) [inline extension/content.css:17]: Why the !important ?
IIRC it prevents the fading animation from working, but not sure here.

File: extension/lib/doc.js [hunk 2]
```diff
@@ -106,6 +106,13 @@ class Doc {
                 let start = sentenceBoundary.index;
                 let end = start + sentenceBoundary.offset;
                 this.sentences.push(new SentencePointer(container_id, start, end));
+                let sentenceId = this.sentences.length - 1;
```
Comment (sjonany) [inline extension/lib/doc.js:109]: Very nice approach man :)))))
Love it. Makes this CL simple.
-- Feel free to resolve.

**Comment by sjonany** [comment]
Oh damn this CL turned out to be much cleaner than I expected.
Very nicely done :D 

LGTM

Just that comment about !important, but eh we will be changing the fading animation later on anyway, so im fine if u squash and merge after leaving a one-line explanation so i understand later.

**Review by sjonany — CHANGES_REQUESTED** [review]

File: extension/lib/doc.js (hunk 2 above)
Comment (benfrancis314) [reply extension/lib/doc.js:109]: Thanks!

File: extension/content.css (hunk 1 above)
Comment (benfrancis314) [reply extension/content.css:17]: Mark causes yellow background, so have all mark classes (which is now every sentence) inherit background !important, so need this to override the mark !important. (See comment in code). Long term solution is to find where the mark defaults are, and nuke them

**Pull request merged**